lib/cts/cases/create.rb

```diff
@@ -450,7 +450,7 @@ def responding_team
         elsif options.key?(:responder)
           responder.responding_teams.first
         else
-          BusinessUnit.includes(:responders).responding.sample
+          BusinessUnit.includes(:responders).responding.active.sample
```

Comment:
Mildly surprising this didn't break an existing spec - not sure this is part of the app we spend much time in!

I wonder if it might be an idea to add some coverage for this change anyway?

Reply:
Changed the existing admin spec to add the same number of deactivated teams as the active ones to cover this scenario (50% chance),  although it doesn't guarantee to test fully, but should be fair chance.

Reply:
Nice! Thanks for adding 👍